src/info.rs

```diff
@@ -489,10 +489,13 @@ impl Info {
 
     fn get_authors(git_history: &[String], n: usize) -> Vec<(String, usize, usize)> {
         let mut authors = std::collections::HashMap::new();
+        let mut names = std::collections::HashMap::new();
         let mut total_commits = 0;
         for line in git_history {
             let commit_author = line.split('\t').collect::<Vec<_>>()[1].to_string();
+            let author_name = line.split('\t').collect::<Vec<_>>()[2].to_string();
```

Comment:
This should probably be called `commit_author_name`

Reply:
Thanks. I think the following variable names are good because of the relationship with the format string of git log.

-  %an = author_name
- %ae = author_email

I've renamed commit_author to author_email in c68e0210